CIP-0124/README.md

```diff
@@ -0,0 +1,222 @@
+---
+CIP: 124
+Title: CIP-25 - Extend token metadata for translations
```

Comment:
```suggestion
Title: Extend token metadata for translations
```
Would be better to keep the long time PR title for now since that's the way reviewers were used to seeing this & there's never been an ambiguity with this being applicable to more modern token types.  And someday it might have a CIP modification that will make it so... therefore to avoid fragmentation it would help to have a more general title.

In any case the other CIP name cannot be in the initial position because it at first glance it appears to be the "wrong" CIP 😅